src/plugins/intel_npu/src/al/include/intel_npu/al/config/options.hpp

```diff
@@ -0,0 +1,1258 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <thread>
+
+#include "intel_npu/al/config/config.hpp"
+#include "npu_private_properties.hpp"
+#include "openvino/runtime/intel_npu/properties.hpp"
+#include "openvino/runtime/internal_properties.hpp"
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+
+namespace intel_npu {
+
+std::string_view stringifyEnum(ProfilingType val);
+std::string_view stringifyEnum(CompilerType val);
+std::string_view stringifyEnum(ElfCompilerBackend val);
+
+}  // namespace intel_npu
+
+}  // namespace ov
+
+namespace intel_npu {
```

Comment:
Not sure here if we need the properties to be part of the `ov::intel_np`u namespace or not. Or if this is set somewhere else.

Also not sure what implications this change can have.